### ISSUE TITLE
Add "linked profiles" 

### ISSUE THREAD

**Issue by palapapa** [gorilla-devs/ferium#333]
### The Problem

Sometimes I want to have several profiles that only differ in output directories and version, but all have the same mods, so I can do `upgrade` with each profile to see which mods I added have compatible versions with that profile. I want to do this because I have a big list of mods I want to play, but some don't support some version and some others don't support some other version. So I want to check which Minecraft version supports the most mods in that list of mods I want to play. Right now the only way to do this is to copy the same profile but change its configuration, but when I add mods to one of those copied profiles, I need to remember to copy the profiles again. 

### Your Solution(s)

Add the ability to link profiles so that the mod lists of them are shared, but can have different configurations.

**Comment by T3sT3ro** [comment]
What you can do:
1. Use one profile, let's name it "common profile"
2. Set it's output directory to one common location, for example `~/Desktop/someModDir/`
3. Create symbolic link named `mods` in both minecraft versions to that common directory. Assuming your you have two minecraft instances: `instances/1.18.2` and `instances/1.19.4` you DELETE the `mods` directory in those two directories, and create symbolic links with `ln -s ~/Desktop/someModDir instances/1.18.2` and `ln -s ~/Desktop/someModDir instances/1.19.4`. With this, both instances will point to the same mods directory
4. install all mods
5. when you want to change the version, use  `ferium configure` to change the profile version and run `ferium upgrade`. This way both instances will see updates to mods when you launch them.

Symbolic links are just pointers to an actual file (or directory).

**Comment by palapapa** [comment]
Another thing that "linked profiles" can achieve but symlinks can't is that I can have different `user` folders for different Minecraft instances. I need this because if some mods don't allow third-party apps to download them, I will need to download them manually and put them in the `user` folder. Using symlinks forces me to share the `user` folder between versions, but I need different `user` folders to put different versions of those manually-downloaded mods.

**Comment by theRookieCoder** [comment]
> A better way might be to allow profiles to inherit other profiles' mods. Any extra mods in the inheriting profile will be added in addition to the inherited mods.

Yes this is how I planned to implement this feature, I should update the title to reflect this.
